app/src/main/res/navigation/nav_graph.xml

```diff
@@ -150,4 +150,16 @@
             android:name="link"
             app:argType="string" />
     </fragment>
+    <fragment
+        android:id="@+id/splashFragment"
+        android:name="com.devshish.internship.presentation.ui.splash.SplashFragment"
+        android:label="fragment_splash"
+        tools:layout="@layout/fragment_splash" >
+        <action
+            android:id="@+id/action_splashFragment_to_authFragment"
+            app:destination="@id/authFragment" />
+        <action
+            android:id="@+id/action_splashFragment_to_homeFragment"
+            app:destination="@id/homeFragment" />
+    </fragment>
```

Comment:
What will happen if I decide to navigate back from authFragment or mainfragment?